### ISSUE TITLE
Popularity Based Results

### ISSUE THREAD

**Issue by rdswift** [toolboc/musicbrainz-conversation-bot#1]
running the query "who sings Under Pressure" yields a list which does not include Queen.  Perhaps eventually the results could be ordered by popularity by utilizing information from AcousticBrainz (or some other rating source).

**Comment by Freso** [comment]
I don't think AcousticBrainz is a good source for popularity—[ListenBrainz](https://listenbrainz.org/) would be a better one. :)

**Comment by rdswift** [comment]
Right.  That's what I was thinking as well, but for some reason the fingers typed AcousticBrainz.  Sorry about that.  🙂
